conda_build/environ.py

```diff
@@ -345,7 +345,11 @@ def system_vars(env_dict, prefix):
     if 'MAKEFLAGS' in os.environ:
         d['MAKEFLAGS'] = os.environ['MAKEFLAGS']
 
-    d['CPU_COUNT'] = get_cpu_count()
+    if 'CPU_COUNT' in os.environ:
+        d['CPU_COUNT'] = os.environ['CPU_COUNT']
```

Comment:
> Although I am not sure this below in conda-build.

Could you please clarify this, @ocefpaf? I believe "below" was suppose to be "belong". Assuming that is the case, I have some questions.

Why shouldn't this change be here? `CPU_COUNT` is already provided for by `conda-build`. We are merely trying to get some control over that value in case it was set improperly. It is also something that other people struggle with too ( https://github.com/conda/conda-build/issues/784 ). By making this change, this will be one less thing we have to watch out for in recipe review.

In the long run, I don't think any pass through environment variables should be hard-coded in `conda-build` source. Instead we should be able to specify some sort of configuration file that allows one to set what environment variables qualify for automatic pass through. This should cutdown on maintenance burden and increase flexibility. Though this last point is interesting, it is out of scope for this proposal and I do not have the bandwidth to pursue it ATM.